Apollo/DeviceViewers/RefreshViewer.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Markup.Xaml;
+using Avalonia.Interactivity;
+
+using Apollo.Core;
+using Apollo.Devices;
+using Apollo.Elements;
+
+namespace Apollo.DeviceViewers {
+    public class RefreshViewer: UserControl {
+        public static readonly string DeviceIdentifier = "refresh";
+
+        void InitializeComponent(){
+            AvaloniaXamlLoader.Load(this);
+            for (int i = 0; i < 4; i++) Macros[i] = this.Get<CheckBox>($"Macro_{i+1}");
```

Comment:
These don't need names, you can simplify by grabbing MacroStack and then doing some LINQ on its .Children, I think Cast<CheckBox>().ToArray() or something similar will do fine